gallery/src/styles/NxList/NxListClickableLinksExample.tsx

```diff
@@ -11,13 +11,13 @@ import { NxFontAwesomeIcon } from '@sonatype/react-shared-components';
 const NxListClickableLinksExample = () =>
   <ul className="nx-list nx-list--clickable">
     <li className="nx-list__item nx-list__item--link">
-      <a href="#/pages/nx-list" className="nx-list__link selected">
+      <a href="#/pages/nx-list" className="nx-list__link">
         <span className="nx-list__text">nx-list page</span>
         <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
       </a>
     </li>
     <li className="nx-list__item nx-list__item--link">
-      <a href="#/pages/nx-table" className="nx-list__link">
+      <a href="#/pages/nx-table" className="nx-list__link selected">
```

Comment:
Any reason this got moved?

Reply:
> Any reason this got moved?

I felt that the styling in the example was clearer when the selected state was on the second item rather than the first